hudi-flink-datasource/hudi-flink/pom.xml

```diff
@@ -269,7 +269,42 @@
                 </exclusion>
             </exclusions>
         </dependency>
-
+        <dependency>
+            <groupId>javax.transaction</groupId>
+            <artifactId>jta</artifactId>
+            <version>1.1</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>javax.transaction</groupId>
+            <artifactId>javax.transaction-api</artifactId>
```

Comment:
What is the purpose of these 2 jars: `javax.transaction:jta`  `javax.transaction:javax.transaction-api`